.github/dependabot.yml

```diff
@@ -31,9 +31,6 @@ updates:
       AspNetCoreHealthChecks:
         patterns:
           - "AspNetCore.HealthChecks.*"
-      EntityFrameworkCore:
```

Comment:
Is it sufficient to disable these? I would assume that groups are just to create batch update PRs, not to enlist packages. I would have added these in the `ignore` section too.

Reply:
I changed it to be `update-types`, so we only take minor and patch updates. Not major.